rclcpp/include/rclcpp/duration.hpp

```diff
@@ -106,6 +110,27 @@ class RCLCPP_PUBLIC Duration
   Duration
   operator*(double scale) const;
 
+  Duration &
+  operator*=(double scale);
+
+  Duration
+  operator*(const rclcpp::Duration & rhs) const;
+
+  Duration &
+  operator*=(const rclcpp::Duration & rhs);
```

Comment:
I do not really understand what this means and usecase, e.g) seconds = seconds * seconds. having scale is not enough?

Reply:
This is a good point, seconds * seconds = seconds^2, not seconds. I don't have any examples of doing this in my code and we don't have a seconds^2 type. Removed this from the PR.